<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Knut</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="OverviewTree" name="overviewTree">
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="headerHidden">
        <bool>true</bool>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
      </widget>
      <widget class="ContentTree" name="contentTree"/>
      <widget class="TextEdit" name="texteditwidget"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionOpenRecent"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuExtract">
    <property name="title">
     <string>Extract</string>
    </property>
    <addaction name="actionExtractActions"/>
    <addaction name="actionExtractMenus"/>
    <addaction name="actionExtractToolBars"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuExtract"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>	  
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionExtractActions">
   <property name="text">
    <string>Extract Actions...</string>
   </property>
  </action>
  <action name="actionExtractMenus">
   <property name="text">
    <string>Extract Menus...</string>
   </property>
  </action>
  <action name="actionExtractToolBars">
   <property name="text">
    <string>Extract ToolBars...</string>
   </property>
  </action>
  <action name="actionOpenRecent">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open Recent...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>OverviewTree</class>
   <extends>QTreeView</extends>
   <header>overviewtree.h</header>
  </customwidget>
  <customwidget>
   <class>ContentTree</class>
   <extends>QTreeView</extends>
   <header>contenttree.h</header>
  </customwidget>
  <customwidget>
   <class>TextEdit</class>
   <extends>QWidget</extends>
   <header>textedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
